content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,117 @@ title: Configure the Datadog Tracing Library
 type: multi-code-lang
 ---
 
-For information about configuration options for tracing libraries, choose your language:
+This page describes configuration options that are supported across all SDKs. To view these common configuration options, see [Common configuration options](#common-configuration-options).
+
+For configuration options specific to your programming language, choose your SDK from the options below:
 
 {{< partial name="apm/apm-compatibility.html" >}}
 
 <br>
 
 To instrument an application written in a language that does not yet have official library support, see the list of [community tracing libraries][1].
 
+## Common configuration options
+The following configuration options behave consistently across multiple Datadog SDKs:
+
+### Traces
+
+`DD_TRACE_<INTEGRATION>_ENABLED`
+: **Default**: `true` <br>
```

Comment:
I'm not sure this is fully accurate - Java has some experimental integrations disabled by default. We may want to make this clear in the description below

Reply:
Maybe a good opportunity to use that `Caveat` option? - Caveat: In Java, integrations labeled as "beta" or marked as "Disabled" in this [document](https://docs.datadoghq.com/tracing/trace_collection/compatibility/java/?tab=graalvm#integrations) are disabled by default.